web-bluetooth/write-descriptor-async-await.js

```diff
@@ -0,0 +1,61 @@
+var myDescriptor;
+
+async function onReadButtonClick() {
+  let serviceUuid = document.querySelector('#service').value;
+  if (serviceUuid.startsWith('0x')) {
+    serviceUuid = parseInt(serviceUuid);
+  }
+
+  let characteristicUuid = document.querySelector('#characteristic').value;
+  if (characteristicUuid.startsWith('0x')) {
+    characteristicUuid = parseInt(characteristicUuid);
+  }
+
+  try {
+    log('Requesting any Bluetooth Device...');
+    const device = await navigator.bluetooth.requestDevice({
+     // filters: [...] <- Prefer filters to save energy & show relevant devices.
```

Comment:
From what I've experienced, it is easier to play with our samples when I use `acceptAllDevices` as the service is not advertised as you've noted.

My goal is to update our BLE Simulator App to add the ability to write/read this descriptor. See https://github.com/WebBluetoothCG/ble-test-peripheral-android/pull/76
I'm already suggesting it in the top paragraph.